CONTRIBUTING.md

```diff
@@ -182,29 +182,24 @@ tools:
 ## Developing in Docker
 
 We have provided a Dockerfile which helps for isolating build problems, and local development.
-Install [Docker](https://www.docker.com/) for your operating system, clone this repo, then
-run `./scripts/start_container.sh`. This should start a local docker container called `arviz`,
-as well as a [`jupyter`](http://jupyter.org/) notebook server running on port 8888. The
-notebook should be opened in your browser automatically (you can disable this by passing
-`--no-browser`). The repo will be running the code from your local copy of `arviz`,
-so it is good for development.
-
-You may also use it to run the test suite, with
-
-```bash
-$  docker exec -it arviz  bash # logon to the container
-$  cd ~/arviz
-$  . ./scripts/test.sh # takes a while!
-```
-
-This should be quite close to how the tests run on TravisCI.
-
-If the container was started without opening the browser, you
-need the notebook instances token to work with the notebook. This token can be
-accessed with
-
-```
-docker exec -it arviz jupyter notebook list
-```
+Install [Docker](https://www.docker.com/) for your operating system, clone this repo. Docker will generate an environment with your local copy of `arviz` with all the packages in Dockerfile.
+
+### Testing in Docker
+Testing the code using docker consists of executing the same file 3 times (you may need root privileges to run it). First run `./scripts/container.sh --build`. This starts a local docker image called `arviz`. Then run `./scripts/container.sh --clear-cache` and eventually run the tests with `./scripts/container.sh --test`. This should be quite close to how the tests run on TravisCI.
```

Comment:
Ah ok, now that I see this I have one comment.

The cache should be cleared before the container is built, otherwise they get copied into the image and cause failures with pytest because the models from the host file system don't unserialize correctly in the docker container.

Let me know if you like the working below


Testing the code using docker consists of executing the same file 3 times (you may need root privileges to run it). 
First run `./scripts/container.sh --clear-cache`. Then run `./scripts/container.sh --build`.  This starts a local docker image called `arviz`. Finally run the tests with `./scripts/container.sh --test`. This should be quite close to how the tests run on TravisCI.

Reply:
Fixed, thanks.